### PR TITLE
Proxmox: add proxmox_node_info module

### LINKED ISSUE #7686: Proxmox: add module to retrieve node status / usage information
### Summary

I am developing a playbook to provision VMs to a Proxmox cluster, and I would like to be able to deploy each enw VM to the "least loaded" node of the cluster. This module will let me retrieve current load information - specifically CPU and memory usage - which I can then sort on to select the "least loaded" node.

### Issue Type

Feature Idea

### Component Name

proxmox_node_info

### Additional Information

```yaml (paste below)
- name: List existing nodes
  community.general.proxmox_node_info:
    api_host: proxmox1
    api_user: root@pam
    api_password: "{{ password | default(omit) }}"
    api_token_id: "{{ token_id | default(omit) }}"
    api_token_secret: "{{ token_secret | default(omit) }}"
  register: proxmox_nodes
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by jwbernin**
##### SUMMARY

Adds a module to retrieve basic information about Proxmox nodes. Design is based on existing proxmox_domain_info module

Fixes #7686 

##### ISSUE TYPE
- Feature Pull Request
- New Module/Plugin Pull Request

##### COMPONENT NAME

proxmox_node_info

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by jwbernin** [comment]
!component +plugins/modules/proxmox_node_info.py

File: plugins/modules/proxmox_node_info.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: proxmox_node_info
+short_description: Retrieve information about one or more Proxmox VE nodes
```
Comment (felixfontein) [inline plugins/modules/proxmox_node_info.py:15]:
```suggestion
short_description: Retrieve information about one or more Proxmox VE nodes
version_added: 8.2.0
```

File: plugins/modules/proxmox_node_info.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+RETURN = '''
+proxmox_nodes:
+    description: List of Proxmox VE nodes.
+    returned: always, but can be empty
+    type: list
+    elements: dict
+    contains:
+      cpu:
+        description: Current CPU usage in fractional shares of this host's total available CPU
```
Comment (felixfontein) [inline plugins/modules/proxmox_node_info.py:46]: All descriptions should consist out of one or multiple paragraphs with complete sentences.
```suggestion
        description: Current CPU usage in fractional shares of this host's total available CPU.
```
Same below.

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

Please take a look at https://github.com/ansible-collections/community.general/blob/main/CONTRIBUTING.md#creating-new-modules-or-plugins, which contains some information for creating new modules.

**Comment by jwbernin** [comment]
Shoot. That last commit with the HEAD updates was a mistake. I'm going to close this PR and create a new one from a different branch in my fork.

**Pull request closed without merging**